find-files.nix

```
@@ -204,15 +204,17 @@ rec {
   # Finding git config
   #
 
+  home = if lib.inPureEvalMode then _: /nonexistent else import ./home.nix;
```

```suggestion
  home = if lib.inPureEvalMode or false then _: /nonexistent else import ./home.nix;
```

for compatibility with older nixpkgs